packages/fxa-payments-server/src/routes/Checkout/index.tsx

```diff
@@ -47,6 +99,31 @@ export const Checkout = ({
     [plans]
```

Comment:
I know it's not part of this patch, but the selected plan should be based on the plan id and not the list of plans?

Reply:
Not sure I follow, can you elaborate?

Reply:
That's the array of dependencies for `useMemo`, and the dependencies used in that function are `productId`, `planId`, and `plansByProductId`, not `plans`.  I think VS Code should be complaining about it to you as it does to me.

Official docs here: https://reactjs.org/docs/hooks-reference.html#usememo

Reply:
Totally see what you mean now, I wonder why my linter wasn't picking up on that 🤔, just pushed that fix.